### ISSUE TITLE
Version 4.27.0 has not been published on registry.terraform.io

### ISSUE THREAD

**Issue by nikolay** [cloudflare/terraform-provider-cloudflare#3213]
### Confirmation

- [X] My issue isn't already found on the issue tracker.
- [X] I have replicated my issue using the latest version of the provider and it is still present.

### Terraform and Cloudflare provider version

Terraform v1.7.5
on darwin_arm64
+ provider registry.terraform.io/cloudflare/cloudflare v4.26.0

### Affected resource(s)

* `cloudflare/cloudflare` provider

### Terraform configuration files

```hcl
The issue is provider-related.
```

### Link to debug output

Not applicable

### Expected output

Install the latest release without errors.

### Actual output

```
Initializing provider plugins...
- Finding cloudflare/cloudflare versions matching "4.27.0"...
╷
│ Error: Failed to query available provider packages
│
│ Could not retrieve the list of available versions for provider cloudflare/cloudflare: no available releases match the given constraints 4.27.0
╵
```

### Steps to reproduce

1. Pin the `cloudflare/cloudflare` provider in versions.tf to `4.27.0` (the latest release).
2. Run `terraform init -upgarde`

**Comment by nikolay** [comment]
@jacobbednarz How is this a HashiCorp issue when other providers publish just fine, and there's [no active incident](https://status.hashicorp.com/) with Terraform Registy?

As you can see, the last version you published is `4.26.0`: https://registry.terraform.io/providers/cloudflare/cloudflare/latest

**Comment by jacobbednarz** [comment]
the latest version we have published and pushed to the hashicorp pipeline is [v4.27.0](https://github.com/cloudflare/terraform-provider-cloudflare/releases/tag/v4.27.0)
